cnew.py

```diff
@@ -78,6 +78,8 @@ def obtntime():
 def randgene():
     numb = 8
     randstrg = ''.join(secrets.choice("ABCDEF" + "0123456789") for i in range(numb))
+    if not randstrg.isupper():
+        randstrg = randgene()
```

Comment:
The likeliness of having a non-upper substring generated from the variable assignment at line number 80 is very small (almost zero). :wink:

Reply:
Ya that was silly 😅. I'll remove that redundant code 👍

Reply:
Yes. That would be helpful.